frontend/public/components/resource-quota.jsx

```diff
@@ -57,17 +72,35 @@ export const getQuotaResourceTypes = (quota) => {
   return _.keys(specHard).sort();
 };
 
-const getResourceUsage = (quota, resourceType) => {
+const getResourceUsage = (quota, resourceType, namespace, isACRQ = false) => {
```

Comment:
We shouldn't need to pass `isACRQ` around in all these functions if we're passing the `quota` object. You can simply check the obj kind and avoid the extra parameter everywhere.